service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpSend.java

```diff
@@ -112,6 +112,14 @@ public void send(String deviceId, Message message) throws IOException
             this.reactor = Proton.reactor(this);
             // Codes_SRS_SERVICE_SDK_JAVA_AMQPSEND_12_008: [The function shall start the Proton reactor object]
             this.reactor.run();
+            try
+            {
+              this.reactor.run();
+            }
+            catch (Exception e)
+            {
+              System.err.println("Error "+ e.getMessage());
```

Comment:
I have checked and I didnt see any CustomLogger.java under services(com.microsoft.azure.sdk.iot.service) package which uses Log4j.
It is only exist under the com.microsoft.azure.sdk.iot.device package. 
Do you want me to create a Logger class?